charts/alertmanager/templates/statefulset.yaml

```diff
@@ -153,6 +169,11 @@ spec:
             - name: http
               containerPort: 9093
               protocol: TCP
+            {{- if or (gt (int .Values.replicaCount) 1) (.Values.additionalPeers) }}
```

Comment:
Apologies I'm a bit out of the loop. Why are there two ways to set this again? What happens if you define additional peers but replicas is 1 and vice versa?

Reply:
Not sure, just followed the current structure for supporting HA. Values file didn't specify how additional peers could be used.